docs/TRANSLATION_STATUS.md

```
@@ -0,0 +1,283 @@
+- JSON Formatter: Online JSON beautifiers
+
+### External Resources
+- [i18next Documentation](https://www.i18next.com/)
+- [Mozilla L10n Guide](https://mozilla-l10n.github.io/localizer-documentation/)
+- [Google Style Guides](https://developers.google.com/style)
+
+## Contact
+
+For translation questions or to volunteer:
+- Create an issue on GitHub with `[Translation]` tag
+- Email: translations@blawby.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)